pkg/loghttp/labels.go

```diff
@@ -19,6 +19,11 @@ type LabelResponse struct {
 // LabelSet is a key/value pair mapping of labels
 type LabelSet map[string]string
 
+// Map coerces LabelSet into a map[string]string. This is useful for working with adapter types.
+func (l LabelSet) Map() map[string]string {
+	return map[string]string(l)
```

Comment:
Small optimization here: you don't need to cast the type, `return l` is enough in this case.